frontend/src/__tests__/cypress/cypress/support/commands/application.ts

```diff
@@ -133,3 +147,13 @@ Cypress.Commands.add('fill', { prevSubject: 'optional' }, (subject, text, option
   cy.wrap(subject).clear();
   return cy.wrap(subject).type(text, options);
 });
+
+Cypress.Commands.add('pfSwitch', (dataId) => {
+  Cypress.log({ displayName: 'pfSwitch', message: dataId });
+  return cy.findByTestId(dataId).parent();
+});
+
+Cypress.Commands.add('pfSwitchValue', (dataId) => {
+  Cypress.log({ displayName: 'pfSwitchValue', message: dataId });
+  return cy.pfSwitch(dataId).find('[type=checkbox]');
+});
```

Comment:
@christianvogt thoughts?

Reply:
My first thought is by not do `cy.pfSwitch('id').switchValue()...`. I tried a quick solution but I need more experience with custom commands and specifically using typescript to create better types for such commands. I would want `switchValue` to only be available when chaining from `pfSwitch`.

This works for now and we can revisit later.